src/ape/contracts/base.py

```diff
@@ -185,6 +185,19 @@ class ContractEvent:
 
 
 class ContractInstance(AddressAPI):
+    """
+    An instance of a smart contract that you can interact with.
+    After you deploy a contract using the :class:`~ape.api.accounts.AccountAPI.deploy` method,
+    you get back a contract instance.
+
+    Usage example::
+
+        from ape import accounts, project
+
+        a = accounts.load("alias")  # Load an account by alias
+        contract = a.deploy(project.MyContract)  # The result of the deploy is a ContractInstance
```

Comment:
```suggestion
        contract = a.deploy(project.MyContract)  # The result of ``deploy`` is a ContractInstance
```

Reply:
Since this is inside a code block, the tick marks don't get rendered

Reply:
<img width="602" alt="Screen Shot 2021-12-20 at 15 32 28" src="https://user-images.githubusercontent.com/19540978/146865558-a11d50e6-6c62-445e-9fd9-0c04f8813608.png">

Reply:
<img width="707" alt="Screen Shot 2021-12-20 at 21 22 04" src="https://user-images.githubusercontent.com/19540978/146865597-a8ea1609-9af1-475b-8501-1cd370c4f888.png">

Reply:
hmm, maybe something like this then:

```
# The result of 'deploy()' is a ContractInstance
```